DotNetWebApi/Controllers/LocationController.cs

```diff
@@ -0,0 +1,106 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using DotNetWebApi.Models;
+
+namespace DotNetWebApi.Controllers;
+
+[Route("[controller]")]
+[ApiController]
+public class LocationController : ControllerBase
+{
+    private readonly TeddyBearsContext _context;
+
+    public LocationController(TeddyBearsContext context)
+    {
+        _context = context;
+    }
+
+    [HttpGet("Locations")]
```

Comment:
Since there is a base route, `("Locations")` can be removed, otherwise your route will be `api/locations/locations`. This applies to other routes in all controllers too.